Demo/Application/DemoDropInViewController.swift

```diff
@@ -78,6 +78,7 @@ class DemoDropInViewController: DemoBaseViewController, DemoDropInViewDelegate {
             guard let result = result, error == nil else {
                 self.progressBlock?("Error: \(error!.localizedDescription)")
                 print("Error: \(error!)")
+                dropInController.dismiss(animated: true, completion: nil)
```

Comment:
Our UI tests would have caught this issue if we were dismissing the Drop-in when an error was returned back to the merchant.